coordinator/clients/traces-generator-api-client/src/main/kotlin/net/consensys/zkevm/coordinator/clients/TracesGeneratorJsonRpcClientV2.kt

```diff
@@ -41,65 +45,111 @@ class TracesGeneratorJsonRpcClientV2(
 
   data class Config(
     val expectedTracesApiVersion: String,
+    val ignoreTracesGeneratorErrors: Boolean = false,
   )
 
   private var id = AtomicInteger(0)
 
+  private fun createFallbackTracesCountersResponse(): GetTracesCountersResponse {
+    return GetTracesCountersResponse(
+      tracesCounters = TracesCountersV2.EMPTY_TRACES_COUNT,
+      tracesEngineVersion = config.expectedTracesApiVersion,
+    )
+  }
+
+  private fun createFallbackConflatedTracesResponse(
+    startBlockNumber: ULong,
+    endBlockNumber: ULong,
+  ): GenerateTracesResponse {
+    val defaultFileName = "$startBlockNumber-$endBlockNumber.conflated.${config.expectedTracesApiVersion}.lt"
```

Comment:
we should add a label to the name indicating they are fake/empty
```suggestion
    val defaultFileName = "$startBlockNumber-$endBlockNumber.fake-empty.conflated.${config.expectedTracesApiVersion}.lt"
```